.github/workflows/check-active-prs-and-slots.yml

```diff
@@ -0,0 +1,103 @@
+name: Check Active PRs and Slots
+
+on:
+  schedule:
+    # Monday at 12 PM UTC - https://cron.help/#0_12_*_*_MON
+    - cron: "0 12 * * MON"
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+defaults:
+  run:
+    shell: pwsh
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  check-pr-slots:
+    runs-on: ubuntu-latest
+    outputs:
+      slotIDs: ${{ steps.listOfSlots.outputs.slotIDs }}
+      slotsNeedDeletion: ${{steps.comparision.outputs.slotsNeedDeletion }}
+
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v2
+
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v2
+        with:
+          path: ./.github
+
+      - name: Azure CLI - Login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - name: Get list of deployed Slots
+        id: slotList
+        run: |
+          $slots = $(az webapp deployment slot list `
+            --name ${{ env.APP_SERVICE_NAME }} `
+            --resource-group ${{env.AZURE_RESOURCE_GROUP }} `
+            --query '[].name' `
+            --output tsv | `
+            sed 's/pr-//g')
+          echo "slots=$slots" >> $env:GITHUB_OUTPUT
+
+      - name: Get list of active PRs
+        id: prList
+        run: |
+          $active_prs=$(gh pr list --state open | awk '{print $1}')
```

Comment:
the `gh` command can return `--json` and be passed a jq query to shape the pr list as an array

Reply:
done